omnigibson/examples/object_states/particle_applier_remover_demo.py

```diff
@@ -109,12 +63,64 @@ def main(random_selection=False, headless=False, short_exec=False):
         position=[0, 0, 0.98],
     )
 
+    tool_cfg = dict(
+        type="DatasetObject",
+        name="tool",
+        visual_only=True,
+        position=[0, 0.3, 5.0],
+    )
+
+    if modifier_type == "particleRemover":
+
```

Comment:
Please update this, the rest of the file, and all your future PRs to NOT leave empty lines at block beginnings.